app/domain/skills/mappers/skill_mapper.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# require_relative '../../../infrastructure/skill_extractor/init'
```

Comment:
Please remove this commented-out code.

Reply:
Issue solved. Please refer to this line of the [commit](https://github.com/KeroKnows/s-killer/commit/c1d43b9257897b20edca40169e25f234a0879b85#diff-adc1eee32180bfa35f4ca82ddfbdb72d99134ebe927a7f0fdb9117ed3bc2b354L3).